src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -70,7 +70,7 @@ const CityColumnChart = (props) => {
   // so this works for a normal or a stacked chart
   const series = props.indicatorIds.map((indicatorId, i) => ({
     index: props.indicatorIds.length - i, // reverse sort the series (to counteract Highcharts)
-    name: INDICATORS[indicatorId].name,
+    name: INDICATORS[indicatorId].shortDescription,
```

Comment:
Not 100% sure on this either (whether to use short description or name) - will need to review once we have the data sorted and can see what needs to be done.